src/main/java/com/example/EnumDemo/dto/ShowDressDto.java

```diff
@@ -0,0 +1,19 @@
+package com.example.EnumDemo.dto;
+
+import com.example.EnumDemo.enumC.Color;
+import com.example.EnumDemo.enumC.Size;
+import com.example.EnumDemo.enumC.Type;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ShowDressDto {
```

Comment:
The name sounds like a method name, please change it to make it sound like a class name

Reply:
Done

Reply:
Changed ShowDressDto --> DressInfoDto